src/library-authoring/EmptyStates.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/require-default-props */
```

Comment:
FYI, we got rid of this eslint rule. If you rebase on `master` to pull in the latest `frontend-build`, you can get rid of this `eslint-disable` directive :)